tests/integration/autonaming/autonaming_test.go

```diff
@@ -32,21 +44,21 @@ func TestAutonaming(t *testing.T) {
 		Quick:        true,
 		ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
 			assert.NotNil(t, stackInfo.Deployment)
-			assert.Equal(t, 3, len(stackInfo.Deployment.Resources))
+			assert.Equal(t, 4, len(stackInfo.Deployment.Resources))
```

Comment:
I see we bumped up the num of resources to assert on, now that we have an extra resource: the temp namespace. is there anyway we can programmatically get at this number w/o having to hard-code them in and push changes every time we extend the tests?

Reply:
Probably, but I'm going to kick that can down the road for now since it would involve changes across all the integration tests.